packages/core/src/internal/journal/utils/emitExecutionEvent.ts

```diff
@@ -217,6 +217,12 @@ function convertExecutionResultToEventResult(
         error: "Transaction reverted",
       };
     }
+    case ExecutionResultType.STRATEGY_HELD: {
```

Comment:
Sorry for being repetitive, but held shouldn't be considered an error.

Reply:
I have added another event result (held), to allow mapping from held to a non-error result. And wired it through the UI.